proxy/envoy/config.go

```diff
@@ -322,6 +324,20 @@ func buildOutboundRoutes(instances []*model.ServiceInstance, services []*model.S
 	return httpConfigs, tcpConfigs
 }
 
+func buildSslContext(hostname string, context *ProxyContext) *SslContext {
+	mesh := context.MeshConfig
+	serviceAccounts := context.Discovery.GetIstioServiceAccounts(hostname)
+	if mesh.AuthMode == "istio-tls" {
+		return &SslContext {
+			CertChainFile:          mesh.AuthConfigPath + "/cert_chain",
+			PrivateKeyFile:         mesh.AuthConfigPath + "/private_key",
```

Comment:
Why not use `tls.key` and `tls.cert` for consistency with Kubernetes TLS secret file names?